pkg/skuba/actions/cluster/init/assets.go

```diff
@@ -111,4 +112,53 @@ ignore-volume-az=true
 No special operation needs to be performed if you deployed the whole
 infrastructure using the terraform state files provided by SUSE.
 `, "~", "`")
+
+	vSphereReadme = strings.ReplaceAll(`# ~vSphere~ integration
+
+Create a file inside this directory named ~vsphere.conf~, with the [supported contents](https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/cloud_config.md):
+
+You can find a template named vsphere.conf.template inside this directory.
+
+~~~
+[Global]
+insecure-flag = "1"
+user = "<VC_ADMIN_USERNAME>"
+password = "<VC_ADMIN_PASSWORD>"
+[VirtualCenter <VC_IP_OR_FQDN>]
+datacenters = "<VC_DATACENTERS>"
+[Workspace]
+server = "<VC_IP_OR_FQDN>"
+datacenter = "<VC_DATACENTER>"
+default-datastore = "<VC_DATASTORE>"
+resourcepool-path = "<VC_RESOURCEPOOL_PATH>"
+folder = "<VC_VM_FOLDER>"
+[Disk]
+scsicontrollertype = pvscsi
+~~~
```

Comment:
According to https://github.com/kubernetes/cloud-provider-vsphere/blob/075def59bd18e2cfd45222c68c9c61358297ca7a/docs/book/tutorials/k8s-vcp-on-vsphere-with-kubeadm.md.

Should we add below to config template?
```
[Global]
port = "443"
[Network]
public-network = "VM Network"
```

Reply:
Thanks for point out. Added for visibility.